app/controllers/register_controller.rb

```diff
@@ -112,6 +112,6 @@ def create
 
   def initialize_controller
     @data_validator = ValidationHelper::DataValidator.new
-    @registers_client = OpenRegister::RegistersClient.new
+    @@registers_client ||= OpenRegister::RegistersClient.new({ cache_duration: 60 })
```

Comment:
this duration should be read from config.

Reply:
Good point